test/functional/p2p_permissions.py

```diff
@@ -116,7 +116,7 @@ def run_test(self):
         self.nodes[1].assert_start_raises_init_error(["-whitebind=noban@127.0.0.1/10"], "Cannot resolve -whitebind address", match=ErrorMatch.PARTIAL_REGEX)
 
     def check_tx_relay(self):
-        block_op_true = self.nodes[0].getblock(self.nodes[0].generatetoaddress(100, ADDRESS_BCRT1_P2WSH_OP_TRUE)[0])
+        block_op_true = self.nodes[0].getblock(self.nodes[0].generatetoaddress(240, ADDRESS_BCRT1_P2WSH_OP_TRUE)[0])
```

Comment:
Can we do this without segwit?

Reply:
I'm not clear what you're trying to achieve. Do you actually want to change the test, or is the question about functionality we support once we disable SegWit?

Reply:
segwit changes the tx encoding and size, and we have different relay rules than bitcoin, and no segwit, so I fear that this test may pass yet an actual dogecoin use case may have a regression that we don't spot. Again - later is fine too, then we just have to put it on a list somewhere, because a passing test is easily overlooked.